dev/step-action/cache-upload.yaml

```diff
@@ -70,12 +70,12 @@ spec:
       value: $(params.googleCredentialsPath)
```

Comment:
This file should be in the config folder. That's how we do it for every other Tekton project.

Reply:
This can be done in a separate PR.

Reply:
We now have step actions in two folder - tekton and this.

Should be merged to config.

Reply:
May be after productization we can do that. for now we want to keep a dev version which we can use with ko:// and a released version which can be used for stepAction